src/mumble/AudioWizard.cpp

```diff
@@ -356,13 +355,16 @@ int AudioWizard::nextId() const {
 
 void AudioWizard::playChord() {
 	AudioOutputPtr ao = Global::get().ao;
-	if (!ao || aosSource || bInit)
+
+	if (!ao || m_chord || bInit) {
 		return;
-	aosSource = ao->playSample(QLatin1String(":/wb_male.oga"), 1.0f, true);
+	}
+
+	m_chord = ao->playSample(QLatin1String(":/wb_male.oga"), 1.0f, true);
 }
 
-void AudioWizard::restartAudio() {
-	aosSource = nullptr;
+void AudioWizard::restartAudio(bool wizardComplete) {
```

Comment:
I think it would be more clear if the parameter indicated what this really is about: whether or not to restart the chord.
```suggestion
void AudioWizard::restartAudio(bool restartChord) {
```
Note that with this rename, the meaning of the flag flips and therefore all its uses have to be negated